From: Wim Taymans <wtaymans@redhat.com>
Date: Thu, 18 Feb 2021 19:25:04 +0100
Subject: bluez5: set the right volumes on the node

Origin: upstream, https://gitlab.freedesktop.org/pipewire/pipewire/-/commit/930b411075a5c1b0d9ddcfe7dd2dbfdc27baab90
---
 spa/plugins/bluez5/bluez5-device.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/spa/plugins/bluez5/bluez5-device.c b/spa/plugins/bluez5/bluez5-device.c
index 1064ecd..69c7e22 100644
--- a/spa/plugins/bluez5/bluez5-device.c
+++ b/spa/plugins/bluez5/bluez5-device.c
@@ -936,7 +936,7 @@ static int node_set_volume(struct impl *this, struct node *node, float volumes[]
 	spa_pod_builder_add_object(&b,
 			SPA_TYPE_OBJECT_Props, SPA_EVENT_DEVICE_Props,
 			SPA_PROP_channelVolumes, SPA_POD_Array(sizeof(float),
-				SPA_TYPE_Float, n_volumes, volumes),
+				SPA_TYPE_Float, node->n_channels, node->volumes),
 			SPA_PROP_channelMap, SPA_POD_Array(sizeof(uint32_t),
 				SPA_TYPE_Id, node->n_channels, node->channels));
 	event = spa_pod_builder_pop(&b, &f[0]);
