Description: policy-node: don't crash without metadata
Author: Wim Taymans <wtaymans@redhat.com>
Origin: upstream, commit:3673265ae20d7b59e89cad6c5238c232796731b2
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1916716
Last-Update: 2021-03-04

Index: pipewire/src/examples/media-session/policy-node.c
===================================================================
--- pipewire.orig/src/examples/media-session/policy-node.c
+++ pipewire/src/examples/media-session/policy-node.c
@@ -810,6 +810,9 @@ static void refresh_auto_default_nodes(s
 {
 	struct default_node *def;
 
+	if (impl->session->metadata == NULL)
+		return;
+
 	/* Auto set default nodes */
 	for (def = impl->defaults; def->key != NULL; ++def) {
 		struct node *node;
