Index: pipewire/src/daemon/pipewire.conf.in
===================================================================
--- pipewire.orig/src/daemon/pipewire.conf.in
+++ pipewire/src/daemon/pipewire.conf.in
@@ -41,7 +41,7 @@ context.properties = {
 
     ## Properties for the DSP configuration.
     #default.clock.rate          = 48000
-    #default.clock.allowed-rates = [ 48000 ]
+    default.clock.allowed-rates = [ 44100 48000 88200 96000 176400 192000 352800 384000 ]
     #default.clock.quantum       = 1024
     #default.clock.min-quantum   = 32
     #default.clock.max-quantum   = 2048
